conf/pipeline/proteinfold/unsw_katana.config

```diff
@@ -4,14 +4,16 @@ profiles {
             config_profile_contact     = '@jscgh'
             config_profile_description = 'nf-core/proteinfold UNSW Katana profile provided by nf-core/configs'
         }
+
+        apptainer.libraryDir    = "/srv/scratch/sbf-pipelines/proteinfold/singularity"
+        apptainer.cacheDir      = "/srv/scratch/${USER}/.images"
+        singularity.libraryDir  = "/srv/scratch/sbf-pipelines/proteinfold/singularity"
+        singularity.cacheDir    = "/srv/scratch/${USER}/.images"
+
         process {
             executor = 'pbspro'
-            queue = 'submission'
 
-            withLabel:gpu_compute {
-                accelerator         = 1
-                clusterOptions      = { "-l host=k095 -l ngpus=1 -l ncpus=${task.cpus} -l mem=${task.memory.toMega()}mb" }
-            }
+            if (params.use_gpu) { clusterOptions = { "-l select=1:ngpus=1:ncpus=8:mem=125gb" } }
```

Comment:
Don't you want to restrict this to gpu labeled processes still? This way the entire pipelines when launched with `use_gpu` would be run on gpu nodes which it not what you'd want on most sites.

Reply:
We don't have a standardised GPU label yet (surprisingly) @pontus so might not work anyway...